src/Button.js

```diff
@@ -15,7 +16,17 @@ class AriaMenuButtonButton extends React.Component {
     ambManager: PropTypes.object.isRequired
   };
 
-  static defaultProps = { tag: 'span' };
+  static defaultProps = {
+    tag: 'span',
+      focusGroupOptions: {
+        wrap: true,
+        stringSearch: true,
+        keybindings: {
+          next: { key: 'ArrowDown' },
```

Comment:
I only see the next option used, not all these other focusGroupOptions. Let's limit the props to what's used.